contracts/src/executors/Executor.sol

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.8;
+
+import {IExecutor, Action} from "./IExecutor.sol";
+import {flipBit, hasBit} from "../utils/math/BitMap.sol";
+
+/// @notice Simple Executor that loops through the actions and executes them.
+/// @dev This doesn't use any type of permission for execution and can be called by anyone.
+/// Most useful use-case is to deploy as non-upgradeable and call from another contract via delegatecall.
+/// If used with delegatecall, DO NOT add state variables in sequential slots, otherwise this will overwrite
+/// the storage of the calling contract.
+contract Executor is IExecutor {
+    /// @notice The internal constant storing the maximal action array length.
+    uint256 internal constant MAX_ACTIONS = 256;
+
+    // keccak256("osx-commons.storage.Executor")
+    bytes32 private constant ReentrancyGuardStorageLocation =
```

Comment:
Could this value be computed? We are doing this for permission ID's

Reply:
It can but then assembly's `sstore` fails with:

```js
TypeError: Only direct number constants and references to such constants are supported by inline assembly.
```